base/latexrelease.dtx

```diff
@@ -752,6 +750,21 @@ of this package available from CTAN}
 \msg_redirect_name:nnn { hooks } { exists } { none }
 %    \end{macrocode}
 %
+% Now a one-off for |ltcmd.dtx|: we need to make \cs{NewDocumentCommand}
+% not complain on an already existing command, but it has to be done
+% after \cs{NewDocumentCommand}, so this is separate:
```

Comment:
Something wrong in the documentation @PhelypeOleinik

Reply:
Ops.  By brain thought “after \cs{NewDocumentCommand} is defined”, but my fingers ate a few words :)